stories/Templates/AuthorPage/AuthorPage.scss

```diff
@@ -1,14 +1,23 @@
-@import '../../assets/scss/variables';
-@import '../../assets/scss/mixins';
-@import '../../assets/scss/breakpoints';
+@import "../../assets/scss/variables";
+@import "../../assets/scss/mixins";
+@import "../../assets/scss/breakpoints";
```

Comment:
@Ashish-srij Please use the proper linting as it should have single quotes

Reply:
@surbhi90 Done.